src/bugsnag.js

```diff
@@ -410,6 +410,63 @@
     enhance(history, "replaceState", wrapBuilder(buildReplaceState));
   }
 
+  function trackWorkers() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsWorkers", true)) {
+      return;
+    }
+
+    var workers = 0;
+
+    window.Worker = hijackConstructor(window.Worker, function(url) {
```

Comment:
Yeah, I completely agree with you. You didn't see my face when writing this (🙃). Do you mean you're considering creating some docs which will show how to *manually* hijack (for instance - in this case) breadcrumbs into workers?